loading-indicator.common.ts

```diff
@@ -0,0 +1,26 @@
+interface AndroidOptionsI {
```

Comment:
Just curious: is the `I` to denote an interface? IMO I'd prefer without the I and it's up to the user to understand it's an interface and not a class they can use. I think this is how NativeScript do it internally. What are your thoughts on that?